src/styles/variables.css

```diff
@@ -104,6 +104,11 @@
     background: rgba(34, 186, 255, 0.08) !important;
   };
 
+  --button-active: {
+    background: rgba(56, 138, 229, 0.1);
```

Comment:
how to see this style?

Reply:
it's applied for the popover items with `isActive: true`

(for the screenshot below i made header plugin's renderSettings() return popover items)
![image](https://user-images.githubusercontent.com/31101125/184537469-96a011fe-2970-4a9c-89c3-c2b14f709fe9.png)